### PR TITLE
Lot of work on README

### PR DESCRIPTION
**Pull request by nolacoder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,91 @@
...
+In this project we learned the importance of dividing team focus between the front-end and back-end and collaborating between those areas to deliver functionality quickly and reliably. We utilzed MySQL Workbench in order to import our dataset to create a seamless integration with our JS models. With the help of many Node packages such as express, sequelize, and handlebars, we were able to serve up customize HTML templates based on search criteria.
+
+## Installation
+
+In order to use our app, simply go to the following link: https://infinite-river-93616.herokuapp.com/
+
+If by any chance you wish to deploy this app on your local machine, use the following steps (you must have MySQL and Node.js installed on your machine):
+- Clone this repo to your machine
+- Install all dependencies
+- In MySQL source the db schema file
```
Comment (methyl8) [inline README.md:28]: npm run seed to create tables and use mySQL workbench to import csv to movies table

File: README.md [hunk 2]
```diff
@@ -1 +1,91 @@
...
+
+## Usage
+
+When the app is deployed you will begin on the home page. Here you can see the navigation bar, a brief overview of our app, and a Browse Movie Collection button. Click the Browse button to see the catalog.
+
+![GIF of homepage and browse](./assets/Home%20gif.gif)
+
+If you are not logged in, your functionality is limited to perusing the browse page slideshow and viewing a variety of movie details when you click a movie image.
+
+If you wish to get full access to the site, click the Login link in the navigation bar. You will also be redireted to the Login page if you try to perform an action limited to registered users such as viewing the My Collection page, attempting to save a movie to you collection, or performing a customized genre or movie title search.
```
Comment (JBrendon15) [inline README.md:40]: typo on redirected

File: README.md [hunk 3]
```diff
@@ -1 +1,91 @@
...
+
+![GIF of homepage and browse](./assets/Home%20gif.gif)
+
+If you are not logged in, your functionality is limited to perusing the browse page slideshow and viewing a variety of movie details when you click a movie image.
+
+If you wish to get full access to the site, click the Login link in the navigation bar. You will also be redireted to the Login page if you try to perform an action limited to registered users such as viewing the My Collection page, attempting to save a movie to you collection, or performing a customized genre or movie title search. 
+
+![GIF of logging in](./assets/Login%20gif.gif)
+
+When you are logged in you can perform a custom search on the Browse page by using the search forms on the left side of the page. You can use one criteria in a search. You search results are populated in the space to the right.
```
Comment (JBrendon15) [inline README.md:44]: typo: Your search results

File: README.md [hunk 4]
```diff
@@ -1 +1,91 @@
...
+
+When the search results appear, clicking on a result will launch the movie details modal. A variety of details are present to help inform you about the movie. Click the Save to My Collection button at the bottom of the modal to save the movie to your favorites.
+
+![GIF of search result and save to collection](./assets/Save%20to%20favorites.gif)
+
+To view your collection, head over to the My Collection page via the navigation bar. Here we diplay your saved movie collection. You can click on each item and view the movie modal details modal. On this modal, you have the ability to remove the movie from your collection by clicking the Remove from My Collection button. Changes are displayed immediately.
+
+![GIF of My Collection page and removing from collection](./assets/Collection.gif)
+
+When you are all dont with your session, logout by clicking the Logout button in the navigation bar. Your collection will remain intact on your next visit. Enjoy!
```
Comment (JBrendon15) [inline README.md:56]: typo: done

**Review by methyl8 — APPROVED** [review]
ok

**Pull request merged**